content/installation/upgrading-manager.md

```diff
@@ -2,96 +2,113 @@
+  1. [Backup the agents certificates]({{< relref "installation/upgrading-manager.md#backup-agent-certificates" >}}) from the Cloudify Manager.
+  1. [Install the Cloudify Manager]({{< relref "installation/upgrading-manager.md#install-cloudify-manager" >}}) latest version on the host.
+  1. [Restore the snapshot]({{< relref "installation/upgrading-manager.md#restore-snapshot" >}}) of the Cloudify Manager to the host.
+  1. (Optional) [Uninstall the Cloudify Manager]({{< relref "installation/upgrading-manager.md#uninstall-cloudify-manager" >}}) from the original host. In Cloudify 4.2 and below, this is called teardown.
+  1. [Migrate agents]({{< relref "installation/upgrading-manager.md#migrate-agents" >}}) to the new Cloudify Manager.
 
 {{% gsNote title="Web interfaces" %}}
 Cloudify Composer and the Web UI are restored to the snapshot state if the snapshot is from a Cloudify Manager 4.2. If you made changes to the Cloudify Manager components, such as creating blueprints in composer or adding widgets to Stage, contact Cloudify Support before you restore the snapshot.
 {{% /gsNote %}}
 
-{{% gsNote title="Premium users" %}}
+{{% gsNote title="Premium Users" %}}
 We recommend that premium users contact Cloudify Support for additional assistance with upgrade and migration.
 {{% /gsNote %}}
 
-  By default, the certificate folder to backup, is: `/etc/cloudify/ssl`.
+# Upgrade Procedures
+
+## Backup a Snapshot
+
+To backup a snapshot of the Cloudify Manager and all of its data:
+
+1. Login to the Cloudify Manager with Cloudify CLI from a remote host.
+1. To create the snapshot, run: ```cfy snapshot create```<br>
+
+  The snapshot is saved to the current tenant. The output of the command shows the snapshot ID. For example, snapshot_XLHCNV.
+1. To download the snapshot, run: ```cfy snapshot download <snapshot_ID>```
+  The snapshot is saved on the Cloudify CLI host.
 
-1. (For in-place upgrade) Make sure that the Cloudify Manager is completely removed from the machine.
+For more about the snapshots command, go to: [snapshots]({{< relref "cli/snapshots.md" >}}).
 
-  * For Cloudify Manager 4.0.0, run these commands on the Cloudify Manager virtual instance to download the teardown script and run it as `sudo`:
-      ```curl -o ~/cfy_teardown_4_0_0.sh https://raw.githubusercontent.com/cloudify-cosmo/cloudify-dev/master/scripts/cfy_teardown_4_0_0.sh```<br>
+## Backup Agent Certificates
 
-      ```sudo bash cfy_teardown_4_0_0.sh```. You must supply an -f flag.<br>
-      For high availability clusters, you must also run ```https://github.com/cloudify-cosmo/cloudify-dev/blob/master/scripts/delete_cluster_4_0_1.py```.
+To backup agent certificates:
 
-1. (Optional) We recommend that you run this command to remove the profile directory of this Cloudify Manager from your local `~/.cloudify/profiles` directory:
-   ```rm -rf ~/.cloudify/profiles/{{ your Manager’s IP address }}```
+* Backup the `/etc/cloudify/ssl` directory to a remote host.
```

Comment:
This is only necessary on an in-place 4.0.0 upgrade to the best of my knowledge.